migrations/1719492374845-CreateRoleTable.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CreateRoleTable1719492374845 implements MigrationInterface {
+    name = 'CreateRoleTable1719492374845'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`CREATE TABLE "roles" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), "name" character varying NOT NULL, CONSTRAINT "PK_c1433d71a4838793a49dcad46ab" PRIMARY KEY ("id"))`);
```

Comment:
I've just realised that you can create enum for roles names for db as well and use it for name property instead of character type 
something like
CREATE TYPE "public"."roles_name_enum" AS ENUM(
            'admin',
            'user'
        )
        
        And then use enum as a type for property 
        "name" "public"."roles_name_enum" NOT NULL 
        
  Probably syntax is a little bit different